tox.ini

```diff
@@ -10,12 +11,10 @@ deps = -r{toxinidir}/requirements.txt
 commands = find {toxinidir} -type f -name "*.py[c|o]" -delete
 
 [testenv:black]
-install_command = pip install {opts} {packages}
```

Comment:
why did these get removed?

Reply:
it's now just by default, so I don't see any value in explicitly adding that.